crates/benches/wasm_opcodes.rs

```diff
@@ -406,13 +406,13 @@ impl Display for WatBuilder {
     }
 }
 
-// Use singlepass compiler (the same one used in protocol) to prevent
+// Use Cranelift compiler (the same one used in protocol) to prevent
```

Comment:
cranelift might actually do a couple of optimizations; nothing too aggressive, but probably enough to optimize things like `loop {}`